packages/app/src/cli/models/extensions/specifications/function.ts

```diff
@@ -25,6 +25,7 @@ const FunctionExtensionSchema = BaseSchema.extend({
       .optional(),
     path: zod.string().optional(),
     watch: zod.union([zod.string(), zod.string().array()]).optional(),
+    wasm_opt: zod.boolean().optional().default(true),
```

Comment:
If I have extension config with this option, and an old CLI, does the old CLI crash on build? Or ignore it?